src/libraries/Microsoft.VisualBasic.Core/tests/CompilerServices/VersionedTests.cs

```diff
@@ -132,6 +142,13 @@ public static IEnumerable<object[]> TypeName_TestData()
             yield return new object[] { (int?)0, "Integer" };
         }
 
+        public static IEnumerable<object[]> TypeName_ComObject_TestData()
```

Comment:
I haven't looked at this PR/issue but note that our tests have to pass on Windows Nano. I am not sure how much of the  COM infrastructure is supported on that SKU and I'm willing to bet that it may not have registrations for these types.

If so you will haev to use something like
`[ConditionalTheory(typeof(PlatformDetection), nameof(PlatformDetection.IsWindows), nameof(PlatformDetection.IsNotWindowsNanoServer))] `
or
`[ConditionalTheory(typeof(PlatformDetection), nameof(PlatformDetection.IsWindows), nameof(PlatformDetection.HasWindowsShell))] `

Reply:
If you have to reset CI again I suggest the latter one.

Reply:
Understood.

Reply:
I have created a sample application to run on a Windows Nano and there is `MSXML2.DOMDocument.6.0`. Do you think that is worth try with this type? Or just keep the test disabled as you suggested?

Reply:
I'd just keep it disabled. (I'm more interested in whether we're missing any coverage at all of stuff like Type.GetTypeFromProgID )